source/fundamentals/indexes.txt

```diff
@@ -144,13 +150,48 @@ created above.
    :language: js
    :start-after: begin-query
    :end-before: end-query
+   :dedent:
 
 Multikey indexes behave differently from non-multikey indexes in terms of
 query coverage, index bound computation, and sort behavior. For a full
 explanation of multikey indexes, including a discussion of their behavior
 and limitations, refer to the :manual:`Multikey Indexes page
 </core/index-multikey>` in the MongoDB manual.
 
+Clustered Indexes
+~~~~~~~~~~~~~~~~~
+
+**Clustered indexes** are indexes that improve the performance of
+insert, update, and delete operations on **clustered collections**.
+Clustered collections store documents ordered by the clustered index key
+value.
+
+To create a clustered index, specify the ``clusteredIndex`` option in
+the ``CollectionOption``. The ``clusteredIndex`` option must specify the
+``_id`` field as the key, and the unique field as ``true``.
+
+The following example uses the ``collection()`` method to create a
+clustered index on the ``_id`` field in the ``vendors`` collection of the
+``tea`` database.
+
+.. code-block:: javascript
+   
+   const db = client.db('tea');
+   const coll = db.collection(
+     'vendors',  
+     {
+       clusteredIndex: { 
+         key: { _id: 1 }, 
+         unique: true 
+       }
+     }
+   );
+
+To learn more, see :v5.3:`Clustered Indexes
+</reference/method/db.createCollection/#std-label-db.createCollection.clusteredIndex>`
+and :v5.3:`Clustered Collections
+</core/clustered-collections/#std-label-clustered-collections>`.
```

Comment:
Based on [this info](https://github.com/mongodb/docs-node/pull/328/files#r903711457), I think we can use the Manual upcoming refs instead of using this directive by including the appropriate objects. Let me know if that makes sense.

Reply:
So I just updated to also have the upcoming in the snooty file, but the link to manual doesn't work. I tested manual before and it didn't work because it isn't part of the link

Reply:
well manual doesn't work because the link doesn't have a /manual in them, but the refs you mentioned below work!